tooling/ast_fuzzer/examples/sample_comptime.rs

```diff
@@ -0,0 +1,27 @@
+//! Print a random comptime AST
+//!
+//! ```shell
+//! cargo run -p noir_ast_fuzzer --example sample_comptime
+//! ```
+use arbitrary::Unstructured;
+use noir_ast_fuzzer::{
+    Config, DisplayAstAsNoirComptime, arb_program_comptime,
+};
+use rand::RngCore;
+
+fn main() {
+    let data = {
+        let mut rng = rand::thread_rng();
+        let mut data = [0u8; 1024 * 1024];
+        rng.fill_bytes(&mut data);
+        data
+    };
+    let mut u = Unstructured::new(&data);
+
+    let mut config = Config::default();
+    config.include_brillig = false;
```

Comment:
Does this have to be disabled? If so, should it be set by `arb_program_comptime` instead?